### PR TITLE
Feature/dropdown nav bar

### PR DESCRIPTION
**Pull request by karthshen**
### Changes
- Implemented the dropdown navigation bar for every pages
- Added a temporary bookme button

### Testing
- `npm install && npm start` to run a local build of the website

### CODE REVIEW

File: src/components/Navbars/dropdownNavbar.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            <NavDropdown title="Pages" id="basic-nav-dropdown">
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Summary-Page"}>Styles</NavDropdown.Item>
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Portrait-Page"}>Portrait</NavDropdown.Item>
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Landscape-Page"}>Landscape</NavDropdown.Item>
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Street-Page"}>Street</NavDropdown.Item>
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Star-Page"}>Star</NavDropdown.Item>
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Urban-Page"}>Urban</NavDropdown.Item>
+              <NavDropdown.Item href={Utility.DEFAULT_URL + "/Wedding-Page"}>Wedding</NavDropdown.Item>
+            </NavDropdown>
+            <Button variant="transparent" href={Utility.DEFAULT_URL + "/Bookme-Page"}>Book me</Button>{' '}
```
Comment (karthshen) [inline src/components/Navbars/dropdownNavbar.js:87]: The bookme button is not needed in this merge. Remove it and save it for the next merge.

File: src/components/Navbars/dropdownNavbar.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+          </button>
+        </div>
+        <Collapse
+          className="justify-content-end"
+          navbar
+          isOpen={navbarCollapse}
+        >
+          <Nav navbar>
+            {/* <NavItem>
+              <NavLink
```
Comment (karthshen) [inline src/components/Navbars/dropdownNavbar.js:110]: Remove unnecessary comments of codes

File: src/components/Navbars/dropdownNavbar.js [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+              <Button
+                className="btn-round"
+                color="danger"
+                href="#pablo"
+                target="_blank"
+                disabled
+              >
+                Upgrade to Pro
+              </Button>
+            </NavItem> */}
```
Comment (karthshen) [inline src/components/Navbars/dropdownNavbar.js:174]: Same as above comment

File: package.json [hunk 4]
```diff
@@ -4,20 +4,29 @@
...
     "react-photo-gallery": "^8.0.0",
     "react-router": "5.0.1",
     "react-router-dom": "5.0.1",
     "react-scripts": "3.0.1",
-    "reactstrap": "8.0.0"
+    "reactstrap": "8.0.0",
+    "styled-components": "^5.1.1"
   },
   "scripts": {
     "start": "react-scripts start",
```
Comment (karthshen) [inline package.json:32]: ### Suggestions on art-style changes
1. Move the `Home` and `Bookme` buttons from navbar to the `Pages` dropdown
2. Rename the `Pages` dropdown to something else for more appropriate functionalities.

File: src/components/Navbars/dropdownNavbar.js (hunk 1 above)
Comment (conelson4) [reply src/components/Navbars/dropdownNavbar.js:87]: Deleted the bookme button. I moved the bookme link into the dropdown navbar.

File: src/components/Navbars/dropdownNavbar.js (hunk 2 above)
Comment (conelson4) [reply src/components/Navbars/dropdownNavbar.js:110]: removed

File: src/components/Navbars/dropdownNavbar.js (hunk 3 above)
Comment (conelson4) [reply src/components/Navbars/dropdownNavbar.js:174]: removed

File: package.json (hunk 4 above)
Comment (conelson4) [reply package.json:32]: 1. moved
2. Pages will be renamed to Menu

File: src/components/Routes/Routes.js [hunk 5]
```diff
@@ -11,6 +11,7 @@ import StreetPage from 'views/pages/StreetPage'
 import UrbanPage from 'views/pages/UrbanPage'
 import WeddingPage from 'views/pages/WeddingPage'
 import ProfilePage from 'views/pages/ProfilePage'
+import BookMePage from 'views/pages/BookMePage'
```
Comment (karthshen) [inline src/components/Routes/Routes.js:14]: This will fail the deployment, as it is not existing yet. I fixed it for you.

**Pull request merged**